src/main/java/underdogs/devbie/question/controller/QuestionController.java

```diff
@@ -49,20 +51,11 @@
 
     @NoValidate
     @GetMapping
-    public ResponseEntity<QuestionResponses> readAllOrderBy(
-        @RequestParam(value = "orderBy", required = false) OrderBy condition
+    public ResponseEntity<QuestionResponses> readAll(
+        @Valid @ModelAttribute QuestionReadRequest questionReadRequest,
+        @Valid @ModelAttribute QuestionPageRequest questionPageRequest
```

Comment:
```suggestion
    public ResponseEntity<QuestionResponses> readAll(
        @Valid QuestionReadRequest questionReadRequest,
        @Valid QuestionPageRequest questionPageRequest
```
ModelAttribute은 빼도 잘 동작하겠네요. 또 Valid도 실상사용하고 있진 않는데요. 이것도 빼는게 좋을까요?

Reply:
물론 제 코드도 Valid는 사용하지 않습니다ㅠㅠ